sdk/lib/3rdparty/freetype/src/cid/cidgload.c

```diff
@@ -342,11 +359,9 @@
     CID_GlyphSlot  glyph = (CID_GlyphSlot)cidglyph;
     FT_Error       error;
 #ifdef __REACTOS__
-    T1_DecoderRec *decoder = malloc(sizeof(T1_DecoderRec));
-    if (!decoder) return FT_Err_Out_Of_Memory;
-/* Ugly but it allows us to reduce the diff */
-#define decoder (*decoder)
-    {
+    T1_DecoderRec *decoder = malloc(sizeof(*decoder));
```

Comment:
Same here, needs the NULL check + early return.

Reply:
Added.